lua/starfall/libs_cl/material.lua

```diff
@@ -718,7 +719,7 @@ end
 -- If the texture in key is not set to a rendertarget, a rendertarget will be created and used.
 -- @param string key The key name to set. $basetexture is the key name for most purposes.
 -- @param string url The url or base64 data
--- @param function? cb An optional callback called when image is loaded. Passes nil if it fails or Passes the material, url, width, height, and layout function which can be called with x, y, w, h to reposition the image in the texture
+-- @param function? cb An optional callback called when image is loaded. Passes nil if it fails or Passes the material, url, width, height, and layout function which can be called with x, y, w, h, [pixelated] to reposition the image in the texture and change the scaling interpolation
```

Comment:
Can you explain that pixelated is a boolean and that enabling it sets the image interpolation to 'nearest'